permissions/plugin-pipeline-maven.yml

```diff
@@ -2,5 +2,7 @@
 name: "pipeline-maven"
 paths:
 - "org/jenkins-ci/plugins/pipeline-maven"
+- "org/jenkins-ci/plugins/pipeline-maven-parent"
```

Comment:
I'm changing it but you were faster than me ;) thanks for pointing it out, pushed.